src/formatDate.js

```diff
@@ -50,7 +50,66 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const dateArr = date.split(fromSeparator);
+  let year = '';
+  let month = '';
+  let day = '';
```

Comment:
you could just declare all three without initializing:
```suggestion
  let year, month, day;
```